ReactiveCocoa/Swift/Signal.swift

```diff
@@ -865,10 +854,10 @@ extension SignalType {
 					sendInterrupted(observer)
 				}
 			}
-			
```

Comment:
Can you try to get all these space changes out of the diff? Looks like you might have a different setting.